Sources/StreamChatUI/Resources/Assets.xcassets/chevron.left.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "chevron.backward.png",
```

Comment:
Have you tested how everything looks on versions older than iOS 13?

Reply:
I haven't 😞 but i'm downloading iOS 13 simulator as we speak

Reply:
Tested ✅ iPhone Xs iOS 13.7
|Dark - Recording|Dark - Locked|Light - Recording|Light - Locked|
|:-------------: |:-------------:| -----:|-----:|
|![Simulator Screen Shot - iPhone Xs - 2023-05-09 at 17 54 12](https://github.com/GetStream/stream-chat-swift/assets/472467/e5725ab9-7ade-4a92-b30f-44c8e82172c6)|![Simulator Screen Shot - iPhone Xs - 2023-05-09 at 17 54 04](https://github.com/GetStream/stream-chat-swift/assets/472467/c4a89c9e-6a88-4641-8738-41169c356d75)|![Simulator Screen Shot - iPhone Xs - 2023-05-09 at 17 53 52](https://github.com/GetStream/stream-chat-swift/assets/472467/870f905f-aa0c-402d-b695-4c916db62821)|![Simulator Screen Shot - iPhone Xs - 2023-05-09 at 17 53 44](https://github.com/GetStream/stream-chat-swift/assets/472467/e342cb0f-47ad-4ea6-bf4f-e3b5e3cd8988)|